utils/helpers/formatting/formatting.ts

```diff
@@ -140,6 +140,20 @@ export const formatMonthIntervalFromPartialDate = safe((dPartial: string) => {
   return [`${dPartial}-01`, `${dPartial}-${lastDayOfMonth}`] as const;
 });
 
+/**
+ * Takes a YYYYMM and return a date
+ *
+ * @param dPartial
+ * @returns [dmin, dmax]
+ */
+export const getDateFromYYYYMM = safe((dPartial: string) => {
+  const yyyy = parseInt(dPartial.slice(0, 4), 10);
+  const mm = parseInt(dPartial.slice(4, 6), 10);
+
+  const date = new Date(yyyy, mm, 0);
```

Comment:
How does it work ? As there is no day 0 in a month ?

Reply:
indeed, it should be 1